lib/rules/boolean-prop-naming.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @fileoverview Enforces consistent naming for boolean props
+ * @author Evgueni Naverniouk
+ */
+'use strict';
+
+var has = require('has');
```

Comment:
Thanks. Not all rules were using ES6, wasn't sure which version of Node we're supporting. Will update accordingly.